README.rst

```diff
@@ -145,35 +146,46 @@ Working with the API within a Python program is straightforward both for
 Premium and Enterprise clients.
 
 Our group's python `tweet parser
-library <https://github.com/twitterdev/tweet_parser>`__ is a requirement.
+library <https://github.com/twitterdev/tweet_parser>`__ is a
+requirement.
 
 Prior to starting your program, an easy way to define your secrets will
 be setting an environment variable. If you are an enterprise client,
 your authentication will be a (username, password) pair. If you are a
 premium client, you'll need to get a bearer token that will be passed
 with each call for authentication.
 
-::
+We advocate putting your API info in a yaml file such as this:
 
-    export TWITTER_SEARCH_PW=<password>
-    export TWITTER_SEARCH_ACCOUNT_NAME=<account_name>
-    export TWITTER_SEARCH_BEARER_TOKEN=<token>
+.. code:: .yaml
 
-The other points that you will have to set in the program are your
-endpoint, the api you want to use. There are functions to generate
-correct API endpoints from this info as well as flags to use the
-``counts`` endpoint instead of the regular endpoint.
+
+    twitter_search_api:
+      endpoint: <FULL_URL_OF_ENDPOINT>
+      account: <ACCOUNT_NAME>
+      username: <USERNAME>
+      password: <PW>
+      bearer_token: <TOKEN>
+
+And filling in the keys that are appropriate for your account type.
+Premium users should only have the ``endpoint`` and ``bearer_token``;
+Enterprise customers should have ``account``, ``username``,
+``endpoint``, and ``password``.
+
+Our credential reader will default to expecing this file in
+``"~/.twitter_search.yaml"``, but you can pass the relevant location as
+needed.
 
 The following cell demonstrates the basic setup that will be referenced
-throughout your program's session. Note that any method of storing your
-credentials is valid here; I am using environment variables for ease of
-use.
+throughout your program's session.
 
 .. code:: python
 
     import os
     import json
-    from twittersearch import ResultStream, gen_rule_payload
+    from unidecode import unidecode
```

Comment:
Might be helpful to mention that the unidecode library isn't necessary, but (I assume) is helpful for display in the context of this readme. At least that's what it looks like to me.

Reply:
I removed the unidecode stuff ; I can strip out any unicode from the readme if needed. I agree that it adds some confusion or something in there.